### ISSUE TITLE
Can't scroll vertical category menu on touch devices

### ISSUE THREAD

**Issue by ericrosenbaum** [LLK/scratch-blocks#1058]
### Expected Behavior

The vertical category menu itself should be scrollable when it extends below the screen (i.e. when the screen is short or the list of categories is long).

### Actual Behavior

On iOS and Android, the vertical category menu is not scrollable.

This seems to be because of Blockly.bindEvent (which calls preventDefault on the event) on the categories.

### Steps to Reproduce

Load the [develop branch](https://llk.github.io/scratch-gui/develop/) on a touch device (iOS or Android). View it in landscape mode.

### Operating System and Browser

iOS or Android

**Comment by rachel-fenichel** [comment]
Is that the same as https://github.com/google/blockly/issues/1174?

**Comment by paulkaplan** [comment]
Yes that is the same. I've marked it as blocked on the upstream issue. Looks like the category menu needs a gesture recognizer? Or just to not preventDefault?

**Comment by msintov** [comment]
@paulkaplan Did you find what the fix to this issue will be? Thank you!

**Comment by paulkaplan** [comment]
Not yet unfortunately. I think the answer will be to use a gesture recognizer instead of binding events directly to the categories? Similar to the way the gesture recognizer can tell the difference between mouse down-ing on a block vs. scrolling the flyout?

**Comment by rachel-fenichel** [comment]
What's the plan for making it clear that the category menu is scrollable, now that it's much more likely to exceed the screen height?

![image](https://user-images.githubusercontent.com/13686399/30405619-c4171384-98a1-11e7-82b1-9dc9be325e46.png)


This is what I see on Chrome (when I made the window small for debugging).

**Comment by paulkaplan** [comment]
@carljbowman ^

**Comment by rachel-fenichel** [comment]
I've just fixed the upstream issue.  It should be a similar fix here, though the scratch-style toolbox isn't built on the closure tree.

**Comment by paulkaplan** [comment]
The fix for scrolling the category menu was pulled in from upstream during https://github.com/LLK/scratch-blocks/pull/1157. It is currently possible to scroll the category menu.

There is still an issue with the way the category menu should look with scrollbars, as mentioned by rachel above, but I'll open a new issue for that.
